### PR TITLE
Chore/a shared component

### PR DESCRIPTION
**Pull request by bob-voorhoede**
- fixed icons
- fixed svg imports
- fixed eslint errors
- fixed vue2/vue3 discrepancies (temporary)
- fixed `external-link` import
- two components are now shared between apps
- updated all imports in browser-extension

### CODE REVIEW

File: projects/@shared/components/collapsible-list.vue [hunk 1]
```diff
@@ -38,10 +47,12 @@
 
             <dl class="collapsible-list__table-row collapsible-list__attribute-value">
               <template v-for="(row, rowIndex) in item">
-                <template v-for="(column, i) in row">
+                <div
```
Comment (bob-voorhoede) [inline projects/@shared/components/collapsible-list.vue:50]: Temporary vue2/vue3 discrepancy fix

File: projects/web-app/src/views/link.view.vue [hunk 2]
```diff
@@ -0,0 +1,194 @@
+<template>
+  <panel-section title="Rel">
+    <collapsible-list :data="linkListData_mocked" />
+  </panel-section>
+</template>
+
+<script>
+// import { computed } from 'vue';
```
Comment (Jamerrone) [inline projects/web-app/src/views/link.view.vue:8]: Can be removed right?

File: projects/web-app/src/views/link.view.vue [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+import useHead from '@/composables/use-head';
+import PanelSection from '@shared/components/panel-section';
+import CollapsibleList from '@shared/components/collapsible-list';
+
+export default {
+  setup() {
+    const headData = useHead().data;
+    console.log(headData.value);
```
Comment (Jamerrone) [inline projects/web-app/src/views/link.view.vue:16]: Can we delete this?

**Review by Jamerrone — APPROVED** [review]
👍 2 small comments.

**Pull request merged**